Dynavity/Dynavity/model/Todo.swift

```diff
@@ -1,10 +1,26 @@
 import SwiftUI
 
-struct Todo: Codable {
-    var id = UUID()
+class Todo {
     var label: String
-    var isCompleted = false
+    var isCompleted: Bool
```

Comment:
Toggling `isCompleted` no longer updates the view. I'll fix this in a separate PR to get this PR merged in as soon as possible.

Reply:
Oops, sorry about that!